mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala

```diff
@@ -97,6 +98,7 @@ class MulticlassClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") overrid
     case "recall" => true
     case "weightedPrecision" => true
     case "weightedRecall" => true
+    case "accuracy" => true
```

Comment:
Can this be one big case, like `case "f1" | "precision|" | ...`? right now it just returns true for everything; could this implementation simply be `true` right now?